TSOClient/FSO.Server/config.json

```diff
@@ -35,9 +35,18 @@
 		],
 
 		"tuning":{
-			"top100":{
-				"bonus_per_visitor_hour": 5,
-				"bonus_per_property_top100_rank": 10
+			"bonus":{
+				"property_bonus":{
+					"per_unit": 20,
+					"overrides":{
+						"1": 1500,
```

Comment:
Oops, that per unit was meant to be 10. I was testing the config was picking up changes :/.